backend/api/v1/rollout_service_test.go

```diff
@@ -41,6 +41,60 @@ func TestGetStatementsFromSchemaGroups(t *testing.T) {
 			expectedStatements:       []string{"ALTER TABLE salary_01 ADD COLUMN num INT\n;\n"},
 			expectedSchemaGroupNames: []string{"projects/p2/databaseGroups/g1/schemaGroups/schema group 1"},
 		},
+		{
+			name:              "simple 1",
+			statement:         "ALTER TABLE alldog ADD COLUMN num INT;",
```

Comment:
What if `ALTER TABLE alldog ADD COLUMN allfish INT;`

Reply:
We are doomed😵

Reply:
Unless we use parsers to replace only the "tablename". But that's not our priority.